airbyte-ci/connectors/pipelines/tests/test_tests/test_common.py

```diff
@@ -243,116 +243,3 @@ async def test_params(self, dagger_client, mocker, test_context_ci, test_input_d
         assert set(acceptance_test_step.params_as_cli_options) == {"-ra", "--disable-warnings", "--durations=3"}
         acceptance_test_step.extra_params = {"--durations": ["5"], "--collect-only": []}
         assert set(acceptance_test_step.params_as_cli_options) == {"-ra", "--disable-warnings", "--durations=5", "--collect-only"}
-
-
-class TestCheckBaseImageIsUsed:
```

Comment:
This is now checked in the `connectors_qa` package https://github.com/airbytehq/airbyte/pull/35322